code/datums/gamemode/factions/syndicate/nukeops.dm

```diff
@@ -236,8 +237,8 @@
 /datum/faction/syndicate/nuke_op/proc/prepare_syndicate_leader(var/datum/mind/synd_mind, var/nuke_code)
 
 	leader = synd_mind
```

Comment:
This is where your issue is. `leader` used to be a nukeOps-level variable, and a mind.
It is now a faction-level variable, and a role.

You want `leader =  synd_mind.GetRole(NUCLEAR_OPERATIVE)`.